lib/components/common/Info.tsx

```diff
@@ -1,198 +1,208 @@
-        <Typography variant="body2">
-          Developed by data scientists and professors at University of California San Diego (UCSD).
-        </Typography>
-      </Box>
-
-      <Box sx={styles.body}>
-        <Box>
-          <Typography variant="h4">About CDE</Typography>
-          <Typography>
-            Common Data Element (CDE) are data elements that have been recommended or required by NIH Institutes and Centers and other organizations for use in research and for other purposes.
-          </Typography>
-
-          <Typography>
-            A data element is a basic unit of information (i.e., a variable) collected within a dataset that has a unique meaning and subcategories (data items) of distinct value.
-          </Typography>
-
-          <Typography>
-            Examples of data elements include gender, race, and geographic location. ODC is implementing NIH’s CDEs where appropriate to aid in standardization and harmonization across different datasets.
-          </Typography>
-        </Box>
-
-        <Box>
-          <Typography variant="h4">How does CDE mapping work</Typography>
-          <Typography>
-            Select a CDE that best fit your dataset fields and once that’s been done and you’ve clicked ‘save’, you can then export it to be used across your lab and have data elements that are standardized.
-          </Typography>
-        </Box>
-
-        <Box>
-          <Typography variant="h4">When to map CDE</Typography>
-          <Typography>
-            There are two different ways you can map CDE,
-            <ol>
-              <li>Before acquiring datasets</li>
-              <li>After all your datasets are available</li>
-            </ol>
-          </Typography>
-        </Box>
-
-        <Box>
-          <Typography variant="h4">How to map CDE</Typography>
-          <Typography>
-            Depending on when you’re mapping CDE, start by creating a comprehensive template for your project, and then choose the most relevant CDEs from the options available.
-          </Typography>
-          <Typography>
-            If you're mapping CDEs after acquiring datasets, simply access the dropdown menu to find an extensive list of CDEs, select the ones that align best with your data, and explore suggested CDEs tailored to your dataset.
-          </Typography>
-
-        </Box>
-
-        <Button disableRipple>View the documentation on CDEs</Button>
-
-        <Divider />
-
-        <Box>
-          <Typography>
-            For more information please contact <Link target="_blank" href="cde@ucsd.com">cde@ucsd.com</Link>
-          </Typography>
-        </Box>
-      </Box>
-    </>
-  )
-
-  return (
-    <>
-      <Box sx={styles.backdrop} onClick={handleClose} />
-      <Box sx={styles.wrap}>
-        {InfoContent()}
-      </Box>
-    </>
-  )
+const Info = (props: { setIsInfoOpen: (b: boolean) => void }) => {
+    const {setIsInfoOpen} = props
```

Comment:
No longer gets the necessary function from context but rather from props. The remaining changes are from the linter